packages/dokeysto_camltc/dokeysto_camltc.3.0.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+name: "dokeysto_camltc"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
```

Comment:
Is this still using `jbuilder` or this invocation needs to be updated?

Reply:
Nice catch. I updated it.